dig/chain.json

```diff
@@ -0,0 +1,93 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "dig",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "Dig Chain",
+  "chain_id": "dig-1",
+  "bech32_prefix": "dig",
+  "daemon_name": "digd",
+  "node_home": "$HOME/.digd",
+  "genesis": {
+    "genesis_url":  "https://raw.githubusercontent.com/notional-labs/dig/master/networks/mainnets/dig-1/genesis.json"
+  },
+  "key_algos": [
+    "secp256k1",
+    "eth-secp256k1"
+],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "udig"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/notional-labs/dig",
+    "recommended_version": "MainNet",
+    "compatible_versions": ["MainNet"]
+  },
+  "peers": {
+    "seeds": [
+      {
+        "id": "37b2839da4463b22a51b1fe20d97992164270eba",
+        "address": "62.171.157.192:26656"
+      },
+      {
+        "id": "e2c96b96d4c3a461fb246edac3b3cdbf47768838",
+        "address": "65.21.202.37:6969"
+      }
+    ],
+    "persistent_peers": [
+      {
+        "id": "33f4788e1c6a378b929c66f31e8d253b9fd47c47",
+        "address": "194.163.154.251:26656"
+      },
+      {
+        "id": "64eccffdc60a206227032d3a021fbf9dfc686a17",
+        "address": "194.163.156.84:26656"
+      },
+      {
+        "id": "be7598b2d56fb42a27821259ad14aff24c40f3d2",
+        "address": "172.16.152.118:26656"
+      },
+      {
+        "id": "f446e37e47297ce9f8951957d17a2ae9a16db0b8",
+        "address": "137.184.67.162:26656"
+      },
+      {
+        "id": "ab2fa2789f481e2856a5d83a2c3028c5b215421d",
+        "address": "144.91.117.49:26656"
+      },
+      {
+        "id": "e9e89250b40b4512237c77bd04dc76c06a3f8560",
+        "address": "185.214.135.205:26656"
+      },
+      {
+        "id": "1539976f4ee196f172369e6f348d60a6e3ec9e93",
+        "address": "159.69.147.189:26656"
+      },
+      {
+        "id": "85316823bee88f7b05d0cfc671bee861c0237154",
+        "address": "95.217.198.243:26656"
+      },
+      {
+        "id": "eb55b70c9fd8fc0d5530d0662336377668aab3f9",
+        "address": "185.194.219.128:26656"
+      }
+    ]
+  },
+  "apis": {
+    "rpc": [
+      {
+        "address": "https://65.21.202.37:8001"
```

Comment:
This only works over http, not https.

Can you add an https endpoint please?

Reply:
opps sorry, Let me change that. For now we just launched for a day and do not have a strong server yet, so we dont have many other servers. We will add them very soon when we have one.